feature_engine/creation/combine_with_reference_feature.py

```diff
@@ -65,6 +65,8 @@ class CombineWithReferenceFeature(BaseEstimator, TransformerMixin):
         transformer will return an error if the training or the datasets to transform
         contain missing values.
 
+    drop_original: bool, default=False
+        If True, the original variables to transform will be dropped from the dataframe.
```

Comment:
I would change the wording to:

If True, the original variables will be dropped from the dataframe after their combination.

Reply:
Oh, I just copied and pasted the wording from CyclicalTransformer w/o even reading it. Should we update that wording too?

Reply:
Just here please :)